tools/eslint-config-web3-base/ts-jest.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint', 'jest'],
+	extends: ['./ts', 'plugin:jest/recommended', 'plugin:jest/style'],
+	env: {
+		'jest/globals': true,
+	},
+	rules: {
+		'jest/valid-title': ['error'],
+		'jest/no-conditional-expect': ['error'],
+		'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+		'jest/consistent-test-it': ['error'],
+		'class-methods-use-this': ['error'],
+		'@typescript-eslint/no-magic-numbers': 'off',
+		'@typescript-eslint/unbound-method': 'off',
+		'@typescript-eslint/no-require-imports': 'off',
```

Comment:
Ideally we wouldn't have in `require` imports, yea?

Reply:
It's a tricky one was added because of Jest setup files. Will update the files to use imports.